spelling-allowlist.txt

```diff
@@ -74,4 +74,10 @@ jsonpickle
 dtype
 dataloader
 KDE
+renderes
```

Comment:
I think it's renders (without an e) but not sure what was the meaning

Reply:
it was complaining about this line 
```python
class CustomNotebookRenderer(plotly_renderes.NotebookRenderer):
```
`plotly_renderes` is a plotly submodule 

actually, this class can be deleted but I would like to keep it for now
(anyway it is commented)

Reply:
oh) it is actually an alias, will fix)